packages/jobs/lib/app.ts

```diff
@@ -19,14 +19,18 @@ const logger = getLogger('Jobs');
 
 process.on('unhandledRejection', (reason) => {
     logger.error('Received unhandledRejection...', reason);
+    report(reason);
     // not closing on purpose
 });
 
 process.on('uncaughtException', (err) => {
     logger.error('Received uncaughtException...', err);
+    report(err);
     // not closing on purpose
 });
 
+initSentry({ dsn: envs.SENTRY_DSN, applicationName: envs.NANGO_DB_APPLICATION_NAME, hash: envs.GIT_HASH });
```

Comment:
what if SENTRY_DSN is not set and/or self-hosted setup?

Reply:
it's disabled if dsn is empty, errors.ts:26 👍🏻